connector/servicegraphconnector/README.md

```diff
@@ -116,6 +116,28 @@ datasources:
     version: 1
 ```
 
+## Configuration
+
+The following settings are required:
+
+- `metrics_exporter`: the name of the exporter that this processor will write metrics to. This exporter **must** be present in a pipeline.
+- `latency_histogram_buckets`: the list of durations defining the latency histogram buckets.
+    - Default: `[2ms, 4ms, 6ms, 8ms, 10ms, 50ms, 100ms, 200ms, 400ms, 800ms, 1s, 1400ms, 2s, 5s, 10s, 15s]`
+- `dimensions`: the list of dimensions to add together with the default dimensions defined above.
+
+The following settings can be optionally configured:
+
+- `store`: defines the config for the in-memory store used to find requests between services by pairing spans.
+    - `ttl`: TTL is the time to live for items in the store.
+        - Default: `2s`
+    - `max_items`: MaxItems is the maximum number of items to keep in the store.
+        - Default: `1000`
+- `cache_loop`: the interval at which to clean the cache.
+- `store_expiration_loop`: the time to expire old entries from the store periodically.
+- `virtual_node_peer_attributes`: the list of attributes need to match for building virtual server node, the higher the front, the higher the priority.
+    - Default: `[db.name, net.sock.peer.addr, net.peer.name, rpc.service, net.sock.peer.name, net.peer.name, http.url, http.target]`
+- `metrics_flush_interval`: the interval at which metrics are flushed to the exporter. Metrics are flush on every received batch of traces by default.
```

Comment:
Missing default

Reply:
The default here is noted, `Metrics are flush on every received batch of traces by default.`. Technically the value is set to `0`, which results in the comment here. I've moved the sentence to make it more clear though, let me know if it doesn't make sense. 👍